cfme/tests/infrastructure/test_host.py

```diff
@@ -46,6 +46,24 @@ def host_ips(provider):
     return tuple(ipaddresses)
 
 
+@pytest.fixture(scope='module')
+def create_250_hosts(appliance):
+    script_downloaded = appliance.ssh_client.run_command(
+        "wget -P /var/www/miq/vmdb/ "
+        "https://gist.githubusercontent.com/NickLaMuro/225833358423723ed17ff294415fa6b4/raw/"
+        "f717ccb83f530f653aabe67fe9389164513ef90d/bz_1580569_db_replication_script.rb")
+    assert script_downloaded.success, script_downloaded.output
+
+    create_250_hosts = appliance.ssh_client.run_command(
+        "cd /var/www/miq/vmdb && bin/rails r bz_1580569_db_replication_script.rb")
+    assert create_250_hosts.success
+    yield
+
+    appliance.ssh_client.run_command("rm -f /var/www/miq/vmdb/bz_1580569_db_replication_script.rb")
+    appliance.ssh_client.run_rails_console("[Host].each(&:delete_all)")
+    appliance.delete_all_providers()
```

Comment:
**Question** Why delete all the providers here? Is it because you removed all the hosts being tracked by CFME, even the ones outside of the 250?

Reply:
Yes. I want to do clean up at the end, the script for the test generates 6 providers like ems_01 and so on, it's easy to do by this method